libs/wire-subsystems/src/Wire/UserGroupSubsystem/Interpreter.hs

```diff
@@ -47,6 +47,7 @@ interpretUserGroupSubsystem = interpret $ \case
   CreateGroup creator newGroup -> createUserGroup creator newGroup
   CreateGroupFull managedBy team mbCreator newGroup -> createUserGroupFullImpl managedBy team mbCreator newGroup
   GetGroup getter gid includeChannels -> getUserGroup getter gid includeChannels
+  GetGroupUnsafe tid gid includeChannels -> getUserGroupUnsafe tid gid includeChannels
```

Comment:
Just to check, is the reason to call this unsafe because we don't know nor check if the group is part of the team?